commit e8dcb81bef146e1eb83f6bd9b171719eeb62016f
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-05-23T18:14:00+01:00 (5 months ago)
    
    TOOLS-2261 mountain-gorilla's launch-build script should set CONFIGURE_BRANCHES
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/tools/launch-build b/tools/launch-build
index adf5d9d..7ae7380 100755
--- a/tools/launch-build
+++ b/tools/launch-build
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -113,6 +113,11 @@ case $PROJECT in
 			BUILD_PARAM="${BUILD_PARAM},$(printf '{"name": "PLATFORM_BUILD_FLAVOR", "value": "%s"}' "${PLAT_FLAVOR}")"
 		fi
 		;;
+	"headnode"|"headnode-debug"|"headnode-joyent")
+		if [[ -n "$BRANCH" ]]; then
+			BUILD_PARAM="${BUILD_PARAM},$(printf '{"name": "CONFIGURE_BRANCHES", "value": "bits-branch: %s"}' "${BRANCH}")"
+		fi
+		;;
 esac
 
 if [[ -n $VERBOSE ]]; then
